closure/testing/test/lenient_test.js

```diff
@@ -1,18 +1,14 @@
-goog.setTestOnly();
-goog.require('goog.testing.asserts');
-goog.require('goog.testing.jsunit');
-goog.require('goog.testing.testSuite');
+goog.module('rules_closure.LenientTest');
+goog.setTestOnly('rules_closure.LenientTest');
 
-goog.scope(function() {
-    const Lenient = goog.require('rules_closure.Lenient');
-    
-    goog.testing.testSuite({
-      testMe: function() {
-        // Without "lenient = True" on the lenient_test rule, this line
-        // fails with "ERROR - could not determine the type of this expression"
-        const val = Lenient();
-        assertEquals(val, 123);
-      }
-    });
-    
-});  // goog.scope
+const testSuite = goog.require('goog.testing.testSuite');
+const Lenient = goog.require('rules_closure.Lenient');
+
+testSuite({
+    testMe: function() {
+    // Without "lenient = True" on the lenient_test rule, this line
+    // fails with "ERROR - could not determine the type of this expression"
+    const val = Lenient();
+    assertEquals(val, 123);
```

Comment:
can you pls fix the formatting?

Reply:
And since you are there
```testMe: function() {..}``` --> ```testMe() { ... }```
And 
```on the lenient_test rule``` --> ```on the lenient_test target```

Reply:
Okay, done!